eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -44,5 +44,6 @@ jobs:
         condition: succeededOrFailed()
         inputs:
           mergeTestResults: true
-          testResultsFiles: ${{ parameters.TestResultsFiles }}
-          testRunTitle: 'Live tests for ${{ parameters.ServiceDirectory }} $(DisplayName)'
\ No newline at end of file
+          testRunTitle: 'Live tests for ${{ parameters.ServiceDirectory }} $(DisplayName)'
+          ${{ if ne(parameters.TestResultsFiles, '') }}:
```

Comment:
Why was this condition necessary?

Reply:
I would have thought that if there are no results files then we should probably disable the entire step.